bcipy/helpers/parameters.py

```diff
@@ -13,9 +13,10 @@ class Parameter(NamedTuple):
     """Represents a single parameter"""
     value: Any
     section: str
-    readableName: str
+    name: str
     helpTip: str
-    recommended_values: list
+    recommended: list
+    editable: bool
```

Comment:
I think this improves readability but it's kind of a major change. These parameters files tend to get used for various things long after data is collected (training models, directing the simulator in how to parse the raw data, etc.). We may want to provide a script to upgrade existing parameters files or add a compatibility layer which can read the old format.

Reply:
I am leaning toward a compatibility layer since this will only be used online and we haven't released a new version in awhile. It can be safely ignored like helpTip in many cases.